client/my-sites/site-settings/spam-filtering-settings.jsx

```diff
@@ -59,19 +58,16 @@ const SpamFilteringSettings = ( {
 		className,
 		header = null;
 
-	if (
-		! shouldShowOfferResetFlow() &&
-		! inTransition &&
-		! ( hasAkismetFeature || hasAntiSpam ) &&
-		! isValidKey
-	) {
+	if ( ! inTransition && ! ( hasAkismetFeature || hasAntiSpam ) && ! isValidKey ) {
 		return (
 			<UpsellNudge
-				description={ translate( 'Automatically remove spam from comments and contact forms.' ) }
+				description={ translate(
+					'Save time, get more responses, give your visitors a better experience - all without lifting a finger.'
+				) }
 				event={ 'calypso_akismet_settings_upgrade_nudge' }
 				feature={ FEATURE_SPAM_AKISMET_PLUS }
-				plan={ PLAN_JETPACK_PERSONAL }
-				title={ translate( 'Defend your site against spam! Upgrade to Jetpack Personal.' ) }
+				plan={ PLAN_JETPACK_SECURITY_DAILY }
+				title={ translate( 'Automatically clear spam from comments and forms' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Automatically clear spam from comments and forms.' )` *ES Score: 11*